### PR TITLE
Fixed some grammar and spelling

### PR DESCRIPTION
**Pull request by suchorski**
Fixed some grammar and spelling

- [x] documentation is added or updated

### CODE REVIEW

**Comment by suchorski** [comment]
Made another PR but the "CLA: trivial" seems to be not working

Need to put like this?
`
CLA: trivial

Fixed some grammar and spelling
`

**Comment by t8m** [comment]
You need to exchange these lines in the commit message.

You can use `git commit --amend` and `git push --force` to fix it.

**Comment by suchorski** [comment]
> 



> You need to exchange these lines in the commit message.
> 
> You can use `git commit --amend` and `git push --force` to fix it.

But is that right, doesn't?
[](https://postimg.cc/G4ssWtCz)

**Review by t8m — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ guidelines:
 
  7. For user visible changes (API changes, behaviour changes, ...),
     consider adding a note in [CHANGES.md](CHANGES.md).
-    This could be a summarising description of the change, and could
+    This could be a summarizing description of the change, and could
```
Comment (paulidale) [inline CONTRIBUTING.md:76]: As per our [documentation policy](https://www.openssl.org/policies/technical/documentation-policy.html#language), British English spellings should be used.  This make this change and other similar changes wrong.

**Review by paulidale — CHANGES_REQUESTED** [review]

**Comment by t-j-h** [comment]
And the amount of changes here probably move this beyond trivial IMHO. 
They are indeed simple changes - but once a patch gets beyond a certain size it isn't "trivial" in the sense it needs to be to not require a CLA.

File: CONTRIBUTING.md (hunk 1 above)
Comment (tom-cosgrove-arm) [inline CONTRIBUTING.md:76]: AAUI OpenSSL generally prefers UK spelling, which uses `s`, rather than US spelling, which uses `z`. There are a few of these in this PR; just commenting on this the once.

File: NOTES-WINDOWS.md [hunk 2]
```diff
@@ -195,7 +195,7 @@ MinGW offers an alternative way to build native OpenSSL, by cross compilation.
 
  * It is also possible to build mingw[64] on Linux or Cygwin.
 
-   In this case configure with the corresponding `--cross-compile-prefix=`
+   In this case, configure with the corresponding `--cross-compile-prefix=`
```
Comment (tom-cosgrove-arm) [inline NOTES-WINDOWS.md:198]: As a native English speaker, I would say that many of the `,` changes in this PR are a matter of style, rather than correctness.

File: README-ENGINES.md [hunk 3]
```diff
@@ -53,7 +53,7 @@ user/admin the set of commands and parameter types a given ENGINE
 implementation supports, and for an application to directly feed string
 based input to those ENGINEs, in the form of name-value pairs. This is an
 extensible way for ENGINEs to define their own "configuration" mechanisms
-that are specific to a given ENGINE (eg. for a particular hardware
+that are specific to a given ENGINE (e.g. for a particular hardware
```
Comment (tom-cosgrove-arm) [inline README-ENGINES.md:56]: Personally I prefer `e.g.`, as done here; however note often that using `eg.` can be a matter of style

File: README-FIPS.md [hunk 4]
```diff
@@ -23,7 +23,7 @@ For example, on Unix the final command
 
     $ make install
 
-effectively executes the following install targets
+effectively executes the following installation targets
```
Comment (tom-cosgrove-arm) [inline README-FIPS.md:26]: Well, they are `install_*` targets, and `make install` is idiomatic, so whether they are `install` or `installation` targets could be considered preference, rather than correctness

File: README.md [hunk 5]
```diff
@@ -148,7 +148,7 @@ Support
 =======
 
 There are various ways to get in touch. The correct channel depends on
-your requirement. see the [SUPPORT](SUPPORT.md) file for more details.
+your requirement. See the [SUPPORT](SUPPORT.md) file for more details.
```
Comment (tom-cosgrove-arm) [inline README.md:151]: Nice catch

File: README.md [hunk 6]
```diff
@@ -133,7 +133,7 @@ Wiki
 ----
 
 There is a Wiki at [wiki.openssl.org] which is currently not very active.
-It contains a lot of useful information, not all of which is up to date.
+It contains a lot of useful information, not all of which is up-to-date.
```
Comment (tom-cosgrove-arm) [inline README.md:136]: Yup, this is better

**Comment by mattcaswell** [comment]
> And the amount of changes here probably move this beyond trivial IMHO.

Added the CLA required hold to this PR.

@suchorski Please can you submit one?

https://www.openssl.org/policies/cla.html

**Comment by suchorski** [comment]
> > And the amount of changes here probably move this beyond trivial IMHO.
> 
> 
> 
> Added the CLA required hold to this PR.
> 
> 
> 
> @suchorski Please can you submit one?
> 
> 
> 
> https://www.openssl.org/policies/cla.html

Submit the ICLA by email?

**Comment by tom-cosgrove-arm** [comment]
Might be an idea to split this into separate PRs, e.g. "Fixes for obvious typos", "Fixes for e.g", "Fixes for -ise/-ize", "Add commas", etc - these individually might be trivial

**Comment by mattcaswell** [comment]
> Submit the ICLA by email?

Yes, send a scan of the completed ICLA to [legal@opensslfoundation.org](mailto:legal@opensslfoundation.org)

File: CONTRIBUTING.md (hunk 1 above)
Comment (suchorski) [reply CONTRIBUTING.md:76]: Solved

File: NOTES-WINDOWS.md (hunk 2 above)
Comment (suchorski) [reply NOTES-WINDOWS.md:198]: Reverted back that one

File: README-FIPS.md (hunk 4 above)
Comment (suchorski) [reply README-FIPS.md:26]: Reverted back that too

**Comment by suchorski** [comment]
> Might be an idea to split this into separate PRs, e.g. "Fixes for obvious typos", "Fixes for e.g", "Fixes for -ise/-ize", "Add commas", etc - these individually might be trivial

I can do both, what do you preffer?

**Comment by mattcaswell** [comment]
Having a CLA on file is always preferred over not having a CLA.

**Comment by suchorski** [comment]
> Having a CLA on file is always preferred over not having a CLA.

Going office right now. Will send from there in an hour

**Comment by suchorski** [comment]
> > And the amount of changes here probably move this beyond trivial IMHO.
> 
> Added the CLA required hold to this PR.
> 
> @suchorski Please can you submit one?
> 
> https://www.openssl.org/policies/cla.html

ICLA sent!

File: INSTALL.md [hunk 7]
```diff
@@ -670,7 +670,7 @@ Don't build support for binary Elliptic Curves
 
 ### enable-ec_nistp_64_gcc_128
 
-Enable support for optimised implementations of some commonly used NIST
```
Comment (tmshort) [inline INSTALL.md:673]: This is an acceptable British spelling.

File: INSTALL.md [hunk 8]
```diff
@@ -1005,7 +1005,7 @@ The `ripemd` algorithm is deprecated and if used is synonymous with `rmd160`.
 
     -Dxxx, -Ixxx, -Wp, -lxxx, -Lxxx, -Wl, -rpath, -R, -framework, -static
 
-These system specific options will be recognised and passed through to the
```
Comment (tmshort) [inline INSTALL.md:1008]: This is an acceptable British spelling.

File: README-ENGINES.md [hunk 9]
```diff
@@ -185,7 +185,7 @@ the shared-library ENGINE implementation. If this command succeeds, the
 (copy of the) 'dynamic' ENGINE will magically morph into the ENGINE
 that has been loaded from the shared-library. As such, any control
 commands supported by the loaded ENGINE could then be executed as per
-normal. Eg. if ENGINE "foo" is implemented in the shared-library
```
Comment (tmshort) [inline README-ENGINES.md:188]: Would it be better to replace with "For example," or "For instance,"? "e.g." is typically used in parenthesized expressions (though not exclusively), so it seems odd here.

**Review by tmshort — COMMENTED** [review]
The changes of "s" to "z" (i.e. British to American convention) are not strictly necessary. It may seem weird to those in the US, but it's not wrong.

Not a fan of having sentences start with "E.g.", but there's nothing wrong with it.

File: README-ENGINES.md (hunk 9 above)
Comment (suchorski) [reply README-ENGINES.md:188]: Replaced

**Comment by suchorski** [comment]
> The changes of "s" to "z" (i.e. British to American convention) are not strictly necessary. It may seem weird to those in the US, but it's not wrong.
> 
> 
> 
> Not a fan of having sentences start with "E.g.", but there's nothing wrong with it.

Revisions done.

File: README-ENGINES.md [hunk 10]
```diff
@@ -1,8 +1,6 @@
-Engines
-=======
+# Engines
```
Comment (mspncp) [inline README-ENGINES.md:1]: Please revert this change (multiple locations). It was an explicit choice to use setext headings, not ATX headings, see commit message of 5f8e6c50bd862d6bb0bbd16b4c0b3be841d9cf32.

**Review by mspncp — CHANGES_REQUESTED** [review]

File: README-ENGINES.md (hunk 10 above)
Comment (suchorski) [reply README-ENGINES.md]: Fixed on entire file

**Comment by suchorski** [comment]
Reverted back tome "s" to "z" too

**Review by suchorski — COMMENTED** [review]
Changed made

File: README-ENGINES.md [hunk 11]
```diff
@@ -54,7 +54,7 @@ implementation supports, and for an application to directly feed string
 based input to those ENGINEs, in the form of name-value pairs. This is an
 extensible way for ENGINEs to define their own "configuration" mechanisms
 that are specific to a given ENGINE (eg. for a particular hardware
-device) but that should be consistent across *all* OpenSSL-based
+device) but that should be consistent across _all_ OpenSSL-based
```
Comment (levitte) [inline README-ENGINES.md:57]: !

File: README-ENGINES.md [hunk 12]
```diff
@@ -76,28 +76,28 @@ documentation is provided in. Any complete or incomplete contributions
...
 
 STABILITY & BUG-REPORTS
-=======================
+-----------------------
 
 What already exists is fairly stable as far as it has been tested, but
 the test base has been a bit small most of the time. For the most part,
 the vendors of the devices these ENGINEs support have contributed to the
-development and/or testing of the implementations, and *usually* (with no
+development and/or testing of the implementations, and _usually_ (with no
```
Comment (levitte) [inline README-ENGINES.md:84]: !

File: README-ENGINES.md [hunk 13]
```diff
@@ -76,28 +76,28 @@ documentation is provided in. Any complete or incomplete contributions
...
+devices, also through whatever vendor support channels are available. If
 none of this is possible, or the problem seems to be something about the
 ENGINE API itself (ie. not necessarily specific to a particular ENGINE
 implementation) then you should mail complete details to the relevant
 OpenSSL mailing list. For a definition of "complete details", refer to
 the OpenSSL "README" file. As for which list to send it to:
 
-  * openssl-users: if you are *using* the ENGINE abstraction, either in an
-    pre-compiled application or in your own application code.
+- openssl-users: if you are _using_ the ENGINE abstraction, either in an
```
Comment (levitte) [inline README-ENGINES.md:97]: !

File: README-ENGINES.md [hunk 14]
```diff
@@ -118,7 +118,7 @@ can allow the user/admin (and thus the hardware vendor's support desk
 also) to provide any such input directly to the ENGINE implementation.
 This way, applications do not need to know anything specific to any
 device, they only need to provide the means to carry such user/admin
-input through to the ENGINE in question. Ie. this connects *you* (and
+input through to the ENGINE in question. Ie. this connects _you_ (and
```
Comment (levitte) [inline README-ENGINES.md:121]: !

File: README-ENGINES.md [hunk 15]
```diff
@@ -180,12 +180,12 @@ loaded ENGINE to be discoverable by application code later on using the
 ENGINE's "id". For most applications, this isn't necessary - but some
 application authors may have nifty reasons for using it. The "LOAD"
 command is the only one that takes no parameters and is the command
-that uses the settings from any previous commands to actually *load*
+that uses the settings from any previous commands to actually _load_
```
Comment (levitte) [inline README-ENGINES.md:183]: !

File: README-ENGINES.md [hunk 16]
```diff
@@ -217,14 +217,14 @@ Applications that support the ENGINE API and more specifically, the
 "control commands" mechanism, will provide some way for you to pass
 such commands through to ENGINEs. As such, you would select "dynamic"
 as the ENGINE to use, and the parameters/commands you pass would
-control the *actual* ENGINE used. Each command is actually a name-value
+control the _actual_ ENGINE used. Each command is actually a name-value
```
Comment (levitte) [inline README-ENGINES.md:220]: !

File: README-ENGINES.md [hunk 17]
```diff
@@ -217,14 +217,14 @@ Applications that support the ENGINE API and more specifically, the
...
 pair and the value can sometimes be omitted (eg. the "LOAD" command).
 Whilst the syntax demonstrated in "openssl engine" uses a colon to
 separate the command name from the value, applications may provide
 their own syntax for making that separation (eg. a win32 registry
 key-value pair may be used by some applications). The reason for the
 "-pre" syntax in the "openssl engine" utility is that some commands
-might be issued to an ENGINE *after* it has been initialised for use.
+might be issued to an ENGINE _after_ it has been initialised for use.
```
Comment (levitte) [inline README-ENGINES.md:227]: !

File: README-ENGINES.md [hunk 18]
```diff
@@ -252,7 +252,7 @@ The main advantage of not statically linking ENGINEs and only using
 "dynamic" for hardware support is that any installation using no
 "external" ENGINE suffers no unnecessary memory footprint from unused
 ENGINEs. Likewise, installations that do require an ENGINE incur the
-overheads from only *that* ENGINE once it has been loaded.
+overheads from only _that_ ENGINE once it has been loaded.
```
Comment (levitte) [inline README-ENGINES.md:255]: !

File: README-ENGINES.md [hunk 19]
```diff
@@ -270,40 +270,41 @@ This example will show building the "atalla" ENGINE in the
...
+    previous section. Eg. from the top-level directory, you might try
+
+        apps/openssl engine -vvvv dynamic \
+              -pre SO_PATH:./crypto/engine/dyn_atalla.so -pre LOAD
 
 If the shared-library loads successfully, you will see both "-pre"
 commands marked as "SUCCESS" and the list of control commands
 displayed (because of "-vvvv") will be the control commands for the
-*atalla* ENGINE (ie. *not* the 'dynamic' ENGINE). You can also add
+_atalla_ ENGINE (ie. _not_ the 'dynamic' ENGINE). You can also add
```
Comment (levitte) [inline README-ENGINES.md:307]: !

**Review by levitte — CHANGES_REQUESTED** [review]
Single `*` is just as correct as single `_` to get italics.  Changing them seems unnecessary

File: README-ENGINES.md [hunk 20]
```diff
@@ -76,28 +76,28 @@ documentation is provided in. Any complete or incomplete contributions
 would help make this happen.
 
 STABILITY & BUG-REPORTS
-=======================
+-----------------------
```
Comment (levitte) [inline README-ENGINES.md:79]: I don't see reason for this level change.

Comment (suchorski) [reply README-ENGINES.md:79]: @mspncp asked for that change. Because was defined settextheadings (====) instead of arc headings (# title)

**Comment by suchorski** [comment]
> Single `*` is just as correct as single `_` to get italics.  Changing them seems unnecessary

That wasn't me. Change it back?

File: README-ENGINES.md (hunk 20 above)
Comment (mspncp) [reply README-ENGINES.md:79]: No, I just asked to use `===` resp. `---` underlinings for h1 resp. h2 headers instead of `# ...` and `## ...`. But not to change the header level.

Comment (mspncp) [reply README-ENGINES.md:79]: See https://github.com/openssl/openssl/pull/19262#pullrequestreview-1132360585 (you might have to unhide the resolved post).

Comment (suchorski) [reply README-ENGINES.md]: > See https://github.com/openssl/openssl/pull/19262#pullrequestreview-1132360585 (you might have to unhide the resolved post).

My mistake.
Fixed

**Comment by levitte** [comment]
> > Single `*` is just as correct as single `_` to get italics.  Changing them seems unnecessary
> 
> That wasn't me. Change it back?

Please do

**Comment by suchorski** [comment]
> > > Single `*` is just as correct as single `_` to get italics.  Changing them seems unnecessary
> 
> > 
> 
> > That wasn't me. Change it back?
> 
> 
> 
> Please do

Done

**Review by mspncp — APPROVED** [review]

**Comment by mspncp** [comment]
Editorial Note: @suchorski this PR will be squashed into one, using your first commit message:

```
Fixed some grammar and spelling

CLA: trivial
```
No action is required on your side, it will be done when the commit gets merged.

**Review by levitte — APPROVED** [review]

**Comment by wan7teen-ping** [comment]
First step

Hi I will help you with everything you need but you have to subscribe to
the youtube channel and I will fulfill all your requests

https://youtu.be/aIaqr4eq18g

Pada Sab, 8 Okt 2022, 3:42 PM Matthias St. Pierre ***@***.***>
menulis:

**Comment by suchorski** [comment]
> Editorial Note: @suchorski this PR will be squashed into one commit, using your first commit message:
> 
> 
> 
> ```
> 
> Fixed some grammar and spelling
> 
> 
> 
> CLA: trivial
> 
> ```
> 
> No action is required on your side, it will be done when the commit gets merged.
> 
> 

Nice. I don't know if it is relevant but I sent the ICLA registration. Needs to remove the CLA trivial from commit message?

Thanks!

**Comment by mspncp** [comment]
> I don't know if it is relevant but I sent the ICLA registration. Needs to remove the CLA trivial from commit message?

Thanks for the heads-up, I [force-pushed](https://github.com/openssl/openssl/compare/ebe9409b469e01159af46be6d523cf2ab79a6f52..bb805328e8fb3c91247fa9b18121e7118e1d9105) the squashed commit without the CLA tag.

**Comment by suchorski** [comment]
> > I don't know if it is relevant but I sent the ICLA registration. Needs to remove the CLA trivial from commit message?
> 
> 
> 
> Thanks for the heads-up, I [force-pushed](https://github.com/openssl/openssl/compare/ebe9409b469e01159af46be6d523cf2ab79a6f52..bb805328e8fb3c91247fa9b18121e7118e1d9105) the squashed commit without the CLA tag.

My pleasure. Thanks

**Comment by wan7teen-ping** [comment]
First step

Hi I will help you with everything you need but you have to subscribe to
the youtube channel and I will fulfill all your requests

https://youtu.be/aIaqr4eq18g

Pada Sab, 8 Okt 2022, 10:38 PM Thiago Suchorski ***@***.***>
menulis:

**Comment by mspncp** [comment]
Merged to master in af33b200da8040c78dbfd8405878190980727171. Thank you for your contribution!
